admin/tests/integration/components/users/user-overview_test.js

```diff
@@ -71,7 +71,7 @@ module('Integration | Component | users | user-overview', function (hooks) {
           email: 'john.snow@winterfell.got',
           username: 'kingofthenorth',
           lang: 'fr',
-          locale: 'fr-FR',
+          locale: 'nl-BE',
```

Comment:
Je ne suis pas contre, mais on risque de changer ce test à chaque nouvelle langue que l'on ajoute.
Cela peut déranger certain (désolé ou pas 😈), mais il faudrait une boucle `[ 'fr', 'en', 'fr-BE', 'fr-FR', 'nl-BE'].forEach(...)`

Reply:
oui c'est vrai haha du coup plutôt dans un test unitaire ?

Reply:
Je pense plutôt integration. Je ne crois pas que tu pourras le faire en unitaire.

Reply:
j'ai réécris le test de cette manière et ça passe :

```
[
        {locale: 'en', lang: 'en'}, 
        {locale: 'fr', lang: 'fr'}, 
        {locale: 'fr-BE', lang: 'fr'}, 
        {locale: 'fr-FR', lang: 'fr'}, 
        {locale: 'nl-BE', lang: 'nl'}, 
      ].forEach((expected) => {
        test.only("displays user's information without creation date", async function (assert) {
          // given
          const store = this.owner.lookup('service:store');
          const user = store.createRecord('user', {
            firstName: 'John',
            lastName: 'Snow',
            email: 'john.snow@winterfell.got',
            username: 'kingofthenorth',
            lang: expected.lang,
            locale: expected.locale,
          });
          this.set('user', user);
  
          // when
          const screen = await render(hbs`<Users::UserOverview @user={{this.user}} />`);
          // await this.pauseTest()
  
          // then
          assert.dom(screen.getByText(`Prénom : ${this.user.firstName}`)).exists();
          assert.dom(screen.getByText(`Nom : ${this.user.lastName}`)).exists();
          assert.dom(screen.getByText(`Adresse e-mail : ${this.user.email}`)).exists();
          assert.dom(screen.getByText(`Identifiant : ${this.user.username}`)).exists();
          assert.dom(screen.getByText(`Langue : ${expected.lang}`)).exists();
          assert.dom(screen.getByText(`Locale : ${expected.locale}`)).exists();
          assert.dom(screen.getByText('Date de création :')).exists();
        });
      })
```

j'ai ajouté les langues dans le forEach, mais je ne sais pas si c'est la meilleure manière de faire